relation-graph errors when creating db

I have created an mireot of the `kidney epithelial cell` branch from the `cell ontology`. Here is the `robot` command if interested:
```
robot extract --method mireot \
    --input cl.owl \
    --branch-from-term obo:CL_0002518 \
    --output cl-mireot.owl
```
When I run `semsql make cl-mireot.db`, I receive the following errors which seem to be related to calling `relation-graph`:
```
semsql make cl-mireot.db 
>>>
	  remove -i cl-mireot.owl --axioms "equivalent disjoint annotation abox type" \
	  filter --exclude-terms /opt/homebrew/lib/python3.11/site-packages/semsql/builder//exclude-terms.txt \
	  -o cl-mireot-min.owl
touch cl-mireot-properties.txt
grep -v ^prefix, /opt/homebrew/lib/python3.11/site-packages/semsql/builder/prefixes/prefixes.csv | grep -v ^obo, | perl -npe 's@,(.*)@: "$1"@'  > cl-mireot-prefixes.yaml.tmp && mv cl-mireot-prefixes.yaml.tmp cl-mireot-prefixes.yaml
relation-graph --disable-owl-nothing true \
                       --ontology-file cl-mireot-min.owl\
		        \
                       --output-file cl-mireot-relation-graph.tsv.tmp \
                       --equivalence-as-subclass true \
                       --mode TSV \
		       --prefixes cl-mireot-prefixes.yaml \
	               --output-individuals true \
	               --output-subclasses true \
                       --reflexive-subclasses true && \
	mv cl-mireot-relation-graph.tsv.tmp cl-mireot-relation-graph.tsv
Unrecognized argument: --disable-owl-nothing
Argument --mode: Malformed output mode: TSV
Unrecognized argument: --prefixes
Unrecognized argument: --output-individuals
make: *** [cl-mireot-relation-graph.tsv] Error 1
rm cl-mireot-properties.txt cl-mireot-min.owl cl-mireot-prefixes.yaml
```

FWIW, I can use `rdftab` like so:
```
sqlite3 cl-miriot.db < prefix.sql  # note: needs copy of prefixes.sql
rdftab cl-mireot.db < cl-mireot.owl  
```

cc @balhoff @jamesaoverton

--- a/README.md
+++ b/README.md
@@ -124,7 +124,7 @@ and how to put things in your PATH. It does not require Docker.
 Requirements:
 
 - [rdftab.rs](https://github.com/ontodev/rdftab.rs)
-- [relation-graph](https://github.com/balhoff/relation-graph)
+- [relation-graph](https://github.com/balhoff/relation-graph) `2.3.1` or higher
 
 After installing these and putting both `relation-graph` and `rdftab.rs` in your path:
 

